test/logger_json/formatters/basic_test.exs

```diff
@@ -114,6 +114,24 @@ defmodule LoggerJSON.Formatters.BasicTest do
     assert log["trace"] == "294740ce41cc9f202dedb563db123532"
   end
 
+  test "logs file, line and mfa as metadata" do
+    metadata =
+      capture_log(fn ->
+        Logger.debug("Hello")
+      end)
+      |> decode_or_print_error()
+      |> Map.get("metadata")
+
+    assert metadata |> Map.get("file") |> to_string() =~ "logger_json/formatters/basic_test.exs"
+    assert metadata |> Map.get("line") |> is_integer()
+
+    assert metadata["mfa"] === [
```

Comment:
I think we need a special case for encoding this one so that it looks like an anonymous function, e.g.: `Elixir.LoggerJSON.Formatters.BasicTest."test logs file, line and mfa as metadata"/1`. See here: https://github.com/Nebo15/logger_json/blob/master/lib/logger_json/formatters/google_cloud.ex#L237

Reply:
Indeed, good point. How about now?

Reply:
@AndrewDryga pinging just in case forgotten - please have a look whenever you have time! 🙏